solarforecastarbiter/reports/figures.py

```diff
@@ -56,16 +56,16 @@ def construct_fx_obs_cds(fx_values, obs_values):
 
 def _obs_name(fx_obs):
     # TODO: add code to ensure obs names are unique
-    name = fx_obs.observation.name
-    if fx_obs.forecast.name == fx_obs.observation.name:
+    name = fx_obs.data_object.name
+    if fx_obs.forecast.name == fx_obs.data_object.name:
         name += ' Observation'
```

Comment:
I'm not sure where this shows up, but you don't want + 'Aggregate' for aggregates?